2-With-broker/README.md

```diff
@@ -150,7 +152,8 @@ Broker support is enabled on a per-PublicClientApplication basis. It is disabled
 var app = PublicClientApplicationBuilder
                 .Create(ClientId)
                 .WithBroker()
-                .WithReplyUri(redirectUriOnIos) // $"msauth.{Bundle.Id}://auth" (see step 6 below)
+                // Android: iOS: $"msauth.{Bundle.Id}://auth" (see step 6 below)
+                .WithReplyUri("Redirect URI for android or iOS") 
```

Comment:
>"Redirect URI for android or iOS" [](start = 30, length = 33)

i would put the actual code here. like have a variable above or show how the switch is happening and then do something like `.WithReplyUri(mobileRedirectUri)`

Reply:
@trwalke : the sample should work out of the box in all configurations